apis/core/lib/domain/job/update.ts

```diff
@@ -11,6 +14,23 @@ interface JobUpdateParams {
   store: ResourceStore
 }
 
+function lindasWebQueryLink(cube: NamedNode, version: number, cubeGraph: NamedNode) {
+  const describe = DESCRIBE`?cube`
+    .FROM(cubeGraph)
+    .WHERE`
+      ${cube} ${schema.hasPart} ?cube .
+      ?cube ${schema.version} ${version} .
+    `.build()
+
+  const queryLink = new URL('https://lindas.admin.ch/sparql')
```

Comment:
Put the URL in config variable?

Reply:
We should discuss this once in more detail. I also see that we should migrate some of these fields into the "Organisation Profile" (currently only BAFU). Can we open an issue and just make a pointer here for now?

Reply:
This entire `DESCRIBE` query is very specific. Maybe the most Lindas-specific piece of code we have

Reply:
Extracted into a configuration variable `TRIFID_UI`